test/Error.spec.js

```diff
@@ -60,6 +60,15 @@ describe('Error', () => {
       ).to.be.instanceOf(Error.StripeIdempotencyError);
     });
 
+    it('copies whitelisted properties', () => {
+      const e = new Error.StripeError({
+        charge: 'foo',
+        unknown_prop: 'bar',
+      });
+      expect(e).to.have.property('charge', 'foo');
+      expect(e).not.to.have.property('unknown_prop', 'bar');
```

Comment:
What happens if you try to access a property that _can_ exist but was not returned by the API, e.g. `decline_code` in this example?

Reply:
`undefined`, but I'll express that in the test case. Thanks!
ptal @ob-stripe